mix.exs

```diff
@@ -42,7 +42,8 @@ defmodule Ebaze.MixProject do
       {:phoenix_live_reload, "~> 1.2", only: :dev},
       {:gettext, "~> 0.11"},
       {:jason, "~> 1.0"},
-      {:plug_cowboy, "~> 2.0"}
+      {:plug_cowboy, "~> 2.0"},
+      {:bcrypt_elixir, "~> 2.0"}
```

Comment:
How is bcrypt being utilized with the passwords?

Reply:
787628c80a65fed180d839b99132b8871bcbc5bf